wger/manager/templates/set/formset.html

```diff
@@ -1,73 +1,152 @@
+                    <em>{% trans "Optional" %}</em>
+                </span>
+                {% endif %}
+            </div>
+            {% endif %} {% endif %} {% endfor %}
+        </div>
+
+        {# Form field rows #}
+        <div class="row">
+            {% for i in field %} {% if forloop.first %}
             <div class="col-xs-2" style="text-align: right; padding-left: 0px;">
                 {% trans "Nr." %} {{ forloop.parentloop.counter }}
             </div>
-            {% endif %}
-
-            {% if not i.is_hidden %}
+            {% endif %} {% if not i.is_hidden %}
             <div class="{% cycle 'col-xs-3' 'col-xs-2' %} {% if i.errors %}has-error{% endif %}" style="padding-left: 0px;">
-            {% endif %}
-
-            {#  Field elements #}
-            {% if i|is_checkbox %}
-                {{ i|form_field_add_css:'checkbox' }}
-            {% else %}
-                {{i|form_field_add_css:'form-control'}}
-            {% endif %}
-
-            {% for error in i.errors %}
-                <div {% if i.errors %}class="has-error has-feedback"{% endif %}>
-                <span class="help-block">
-                    <p>{% trans error %}</p>
-                </span>
+                {% endif %} {# Field elements #} {% if i|is_checkbox %} PRINT {{ i|form_field_add_css:'checkbox' }} {% else %} {{i|form_field_add_css:'form-control'}}
+                {% endif %} {% for error in i.errors %}
+                <div {% if i.errors %}class="has-error has-feedback" {% endif %}>
+                    <span class="help-block">
+                        <p>{% trans error %}</p>
+                    </span>
                 </div>
-            {% endfor %}
-
-            {% if not i.is_hidden %}
+                {% endfor %} {% if not i.is_hidden %}
             </div>
-            {% endif %}
-        {% endfor %}
+            {% endif %} {% endfor %}
+        </div>
     </div>
+    {% endfor %}
 </div>
-{% endfor %}
-</div>
+<script>
+    var noSets = document.getElementById('id_sets').value;
+    var value = '';
+    var noReps = '';
+    var id = document.querySelectorAll('input.form-control')[4].id;
+    var exerciseNo = id.replace(/^\D+|\D.*$/g, '');
+    var repsId = '';
+    var weightId = '';
+    var repClass = '';
+    var weightClass = '';
+    var i = null;
+
+    function repetitions() {
+        noReps = document.getElementById('input-1').value;
+        if (noReps < 1) {
+            repClass = 'col-xs-3 has-error';
+        } else {
+            repClass = 'col-xs-3';
+        }
+        document.getElementById('col-xs-3').className = repClass;
+    }
+
+    function weights() {
+        value = document.getElementById('input-2').value;
+        if (value < 30) {
+            weightClass = 'col-xs-4 has-error';
+        } else {
+            weightClass = 'col-xs-4';
+        }
+        document.getElementById('col-xs-4').className = weightClass;
+    }
+
+    function calcDrops() {
```

Comment:
Good implementation of the event handler. However, I think you can improve variable naming, e.g `percentageSet` instead of `percSet`, it is more explicit.

Reply:
@MarloneA thank you @Rodgers-M I will look into that. I think it would make the code more readable